student.rb

```diff
@@ -0,0 +1,14 @@
+require './person'
+
+class Student < Person
+  def initialize(classroom, age, _name: 'Unknown', _parent_permission: true)
+    super(age, _name: 'Unknown', _parent_permission: true)
+    @classroom = classroom
+  end
+
+  def play_hooky
+    "¯\(ツ)/¯"
+  end
+end
+student = Student.new('Hey', 10)
+puts student.can_use_services?
```

Comment:
- [ ] [OPTIONAL] You added this piece of code to your codebase I assume to test your code, please make sure to clean up your code by removing unused pieces of code before sending it for review.